src/leaflike/migrations.clj

```diff
@@ -0,0 +1,23 @@
+(ns leaflike.migrations
+  (require [ragtime.jdbc :as jdbc]
+           [ragtime.repl :as repl]))
+
+(defn db-spec [] {:connection-uri "jdbc:postgresql://localhost:5432/leaflike"})
```

Comment:
Move the connection string into a seperate configuration file using [profiles.clj](https://github.com/technomancy/leiningen/blob/master/doc/PROFILES.md). Doing this allows you to seperate configuration from your app (for dev/testing/prod).

Reply:
You could also take a look at something like https://github.com/yogthos/config.

Reply:
@kirang89 @namratachaudhary I would strongly recommend using something like [cprop](https://github.com/tolitius/cprop) for configuration, instead of tying the configuration to leiningen like `environ` does it. `yogthos/config` seems ok, but `cprop` is more feature rich.